cmd/minishift/cmd/delete.go

```diff
@@ -98,6 +100,28 @@ func removeInstanceConfigs() {
 	}
 }
 
+func removeProfileFromConfig() {
+	name := profileActions.GetProfileName()
+	activeProfile := profileActions.GetProfileName()
```

Comment:
The following looks errorneous:
```
	name := profileActions.GetProfileName()
	activeProfile := profileActions.GetProfileName()

	//If there are multiple profiles present
	if name == activeProfile {
```
this will always result in `true` because the `name` and `activeProfile` would return the same value... unless, side-effects are at play here.